invoiceitem.go

```diff
@@ -12,7 +12,9 @@ type InvoiceItemParams struct {
 	Description  *string `form:"description"`
 	Discountable *bool   `form:"discountable"`
 	Invoice      *string `form:"invoice"`
+	Quantity     *int64  `form:"quantity"`
 	Subscription *string `form:"subscription"`
+	UnitAmount   *int64  `form:"unit_amount"`
 }
```

Comment:
I think you want to add them to the `InvoiceItem` struct too

Reply:
Yes ... that would make sense. I just pushed that change (only `UnitAmount` was missing).